### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by KevinWalkerFan** [mvirgo/MLND-Capstone#23]
Do u have any evaluation code to evaluate the accuracy of the model？Can u share the code?

**Comment by KevinWalkerFan** [comment]
Or do u have the code to calculate the accuracy in train/val?

**Comment by mvirgo** [comment]
The "test" dataset does not have labels, so it is primarily just based off of visuals. As such, there is no way to test the exact accuracy of those. Train/validation accuracy would be printed out by Keras as normal during training.
